flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/catalog/exceptions/TableAlreadyExistException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.api.catalog.exceptions;
+
+import org.apache.flink.table.api.catalog.ObjectPath;
+
+/**
+ * Exception for trying to create a table (or view) that already exists.
+ */
+public class TableAlreadyExistException extends RuntimeException {
+
+	private static final String MSG = "Table (or view) $catalog.$table already exists.";
```

Comment:
The MSG seems to be in scala style. Should be 'Table (or view) %s.%s already exists.'